apps/products-feed/src/pages/api/feed/[url]/[channel]/google.xml.ts

```diff
@@ -16,6 +16,10 @@ import { getDownloadUrl, getFileName } from "../../../../../modules/file-storage
 import { RootConfig } from "../../../../../modules/app-configuration/app-config";
 import { z, ZodError } from "zod";
 
+export const config = {
+  maxDuration: 5,
```

Comment:
```suggestion
  maxDuration: 5 * 60,  // 5 minutes
```

Value here is expected to be in seconds